dependencies.yaml

```
@@ -100,7 +106,7 @@ dependencies:
           - numpydoc
           - pydata-sphinx-theme
           - recommonmark
-          - sphinx
+          - sphinx<6
```

```suggestion
          # TODO: sphinx upper version limit can likely be removed after pydata 0.13
          # is released. See https://github.com/pydata/pydata-sphinx-theme/issues/1094
          - sphinx<6
```

Might not be a bad idea to leave a note here so that we can unpin this in the future.